### PR TITLE
Fix bug in _reset_batch_norm

### PR DESCRIPTION
**Pull request by originlake**
When 'scale' or 'center' is set to False in the BatchNormalization layer, there will be no tensor for gamma or beta or both.
This fix extract layer weights based on layer config

### CODE REVIEW

**Review by alexstoken — APPROVED** [review]
changes solve bug addressed in PR for keras.py. Will need to make similar adjustments to tfkeras.py.

**Comment by alexstoken** [comment]
Hi @originlake , 

Thanks for the PR. I have reproduced the bug you describe, when either 'scale' or 'center' is False. Your PR fixes it well. We will need to make the same changes in tfkeras.py. @simon-larsson , I will make those changes after I merge this PR. 

Thank you @originlake !

**Comment by simon-larsson** [comment]
This has been released in 0.1.5. Thank you, @originlake!
